manifests/b/BepassOrg/OblivionDesktop/0.35.1-beta/BepassOrg.OblivionDesktop.locale.en-US.yaml

```diff
@@ -0,0 +1,81 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: BepassOrg.OblivionDesktop
+PackageVersion: 0.35.1-beta
+PackageLocale: en-US
+Publisher: BepassOrg
+PublisherUrl: https://github.com/bepass-org/oblivion-desktop
+PublisherSupportUrl: https://github.com/bepass-org/oblivion-desktop/issues
+PrivacyUrl: https://github.com/bepass-org/oblivion-desktop?tab=security-ov-file#
+Author: ircfspace
+PackageName: Oblivion Desktop
+PackageUrl: https://github.com/bepass-org/oblivion-desktop
+License: Oblivion Desktop License
+LicenseUrl: https://github.com/bepass-org/oblivion-desktop/blob/HEAD/LICENSE.md
+Copyright: Copyright © 2024 Oblivion Desktop
+CopyrightUrl: https://github.com/bepass-org/oblivion-desktop/blob/main/LICENSE.md
+ShortDescription: Unofficial Warp Client
+Description: Oblivion provides secure, optimized internet access through a user-friendly Windows/Mac/Linux app using cloudflare warp technology
+Moniker: oblivion
+Tags:
+- cloudflare
+- oblivion
+- proxy
+- psiphon
+- vpn
+- warp
+ReleaseNotes: |-
+  نسخه ۰.۳۵.۱ بتا برای ویندوز، لینوکس و مک منتشر شد.
+  برخی‌از تغییرات برنامه:
+  - [x] اضافه‌شدن ویژگی «پروفایل» جهت ذخیره‌سازی اندپوینت‌ها #323
+  - [x] بهبود دسترس‌پذیری برنامه #382 (با تشکر از @GhostByeBye)
+  - [x] رفع مشکل ست‌شدن صحیح پروکسی در gnome #337 (با تشکر از @ParsaImi)
+  - [x] پیروی متن خطاها و منوی SystemTray از فایل زبان #156
+  - [x] رفع مشکل ست‌شدن آیکون برنامه در SystemTray #348 (با تشکر از @morealaz)
+  - [x] رفع مشکل بسته‌نشدن برنامه در macOS #353 (با تشکر از @gdariushahmadi)
+  - [x] تفکیک و بروزرسانی لیست اندپوینت‌های پیشنهادی (با تشکر از amin_o__o و xs_amirparsa)
+  - [x] رفع مشکل اجرای خودکار برنامه در هنگام راه‌اندازی سیستم
+  - [x] ری‌فکتور و بهینه‌سازی کدها (با تشکر از @GhostByeBye)
+  - [x] جلوگیری از نمایش خطای a javascript error occurred #291
+  - [x] ذخیره خودکار اندپوینت در آخرین اتصال موفق
+  - [x] تغییر اعلان داخلی بروزرسانی‌های برنامه به پیش‌شرط Latest
+  - [x] بهبود حرکت بین سربرگ‌ها در بخش تنظیمات
+  - [x] رفع برخی‌از اشکالات جزئی

```

Comment:
Not all of these are en-US localized. Please keep only the en-US localization in the en-US locale file and add additional locale files as needed for other localizations